### ISSUE TITLE
A-Frame inspector style bug

### ISSUE THREAD

**Issue by ben-sharp** [gladeye/aframe-preloader-component#1]
Some bootstrap classes are conflicting causing the A-Frame inspector to break a bit.

**Comment by HAZARDU5** [comment]
Yeah I've encountered this before. Currently you can detect the inspector being open but it requires a bunch of hacky checks to do this.
